examples/flow_benchmark_2d_case_1.py

```diff
@@ -0,0 +1,114 @@
+
+    Inflow on west (left) and prescribed pressure on east (right).
+
+    """
+
+    def bc_values_pressure(self, boundary_grid: pp.BoundaryGrid) -> np.ndarray:
+        """Pressure value of 1 Pa on east side."""
+        bounds = self.domain_boundary_sides(boundary_grid)
+        values = np.zeros(boundary_grid.num_cells)
+        values[bounds.west] = self.fluid.convert_units(1, "Pa")
+        return values
+
+    def bc_type_darcy_flux(self, sd: pp.Grid) -> pp.BoundaryCondition:
+        """Assign Dirichlet to the east boundary. The rest are Neumann by default."""
+        bounds = self.domain_boundary_sides(sd)
+        bc = pp.BoundaryCondition(sd, bounds.east, "dir")
+        return bc
+
+    def bc_values_darcy_flux(self, boundary_grid: pp.BoundaryGrid) -> np.ndarray:
+        """Inflow on the west boundary.
+
+        Per PorePy convention, the sign is negative for inflow and the value is
+        integrated over the boundary cell volumes. Since the inflow boundary contains
+        a fracture, the latter includes the fracture specific volume.
+
+        Parameters:
+            boundary_grid: Boundary grid.
+
+        Returns:
+            Boundary values.
+
+        """
+        bounds = self.domain_boundary_sides(boundary_grid)
+        values = np.zeros(boundary_grid.num_cells)
+        # Inflow on the west boundary. Sign as per PorePy convention.
+        val = self.fluid.convert_units(-1, "m * s^-1")
+        # Integrate over the boundary cell volumes.
+        values[bounds.west] = val * boundary_grid.cell_volumes[bounds.west]
+        # Scale with specific volume.
+        sd = boundary_grid.parent
+        trace = np.abs(sd.cell_faces)
+        specific_volumes = self.specific_volume([sd]).value(self.equation_system)
+        values *= boundary_grid.projection() @ trace @ specific_volumes
+        return values
+
+
+class Permeability(DimensionDependentPermeability):
+    """Tangential permeability specification for Case 1 of the 2D flow benchmark.
+
+    The normal permeability is handled by the `SolidConstants`' `normal_permeability`
+    parameter.
+
+    """
+
+    def fracture_permeability(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
+        """Permeability of fractures.
+
+        Parameters:
+            subdomains: List of subdomains.
+
+        Returns:
+            Cell-wise permeability operator.
+
+        """
+        size = sum([sd.num_cells for sd in subdomains])
+        val = self.params.get("fracture_permeability", 1e-4)
+        permeability = pp.wrap_as_dense_ad_array(
+            self.solid.convert_units(val, "m^2"), size
+        )
+        return self.isotropic_second_order_tensor(subdomains, permeability)
+
+
+class FlowBenchmark2dCase1Model(  # type:ignore[misc]
+    FluxDiscretization,
+    Geometry,
+    Permeability,
+    BoundaryConditions,
+    pp.fluid_mass_balance.SinglePhaseFlow,
+):
+    """Complete model class for case 1 from the 2d flow benchmark."""
```

Comment:
I would also add the following in the model docstring to guide the user on how to set up the two variants of the benchmark:

The benchmark specifies two variants; one with conductive fractures and one with blocking fractures. To set up an instance of the first variant you have to pass the keyword `'fracture_permeability'` with a value `1e4` as a model parameter. To set up an instance of the second variant, set the value to `1e-4` instead.

Reply:
Yes, the proposed solution is way more elegant. I would not include this comment now.